app/controllers/cart_items_controller.rb

```diff
@@ -8,4 +9,12 @@ def create
 		flash[:notice] = "You have #{pluralize(@cart.count_of, "item")} in your cart."
 		redirect_to '/items'
 	end
+
+	def destroy
+		@item = Item.find(params[:id])
+		flash_link = "#{view_context.link_to @item.title, item_path(@item)}"
+		flash[:success] = {color: "green", message: "Successfully removed #{flash_link} from your cart."}
```

Comment:
Line is too long. [99/80]<br>Space inside { missing.<br>Space inside } missing.<br>Tab detected.